CMakeLists.txt

```diff
@@ -54,8 +58,14 @@ target_include_directories(precice-aste-run PRIVATE src thirdparty)
 target_link_libraries(precice-aste-run
   precice::precice
   Threads::Threads
+  Boost::boost
+  Boost::log
+  Boost::log_setup
+  Boost::program_options
+  Boost::system
+  Boost::thread
+  Boost::unit_test_framework
```

Comment:
Is this actually required?

Reply:
Which one? The packages or the CMake setup? unit tests are required.

Reply:
Is linking to the unit test framework required in precice-aste-run?

Reply:
yes